src/unix/aix.c

```diff
@@ -64,11 +64,19 @@
 #define RDWR_BUF_SIZE   4096
 #define EQ(a,b)         (strcmp(a,b) == 0)
 
+static uv_mutex_t mutex;
+static uv_once_t once = UV_ONCE_INIT;
 static void* args_mem = NULL;
 static char** process_argv = NULL;
 static int process_argc = 0;
 static char* process_title_ptr = NULL;
 
+
+static void init_once(void) {
+    uv_mutex_init(&mutex);
```

Comment:
Style: two space indent.  (Also happens in other files.  I won't point them out individually.)